app/controllers/verify/sessions_controller.rb

```diff
@@ -44,6 +44,7 @@ def process_failure
         redirect_to verify_session_dupe_path
       else
         show_warning if step.form_valid_but_vendor_validation_failed?
+        @view_model = SessionsNew.new
```

Comment:
yes because it's a `render` call to `new`, we nee to re-instantiate. Otherwise get `undefined method `title' for nil:NilClass` errors :(